samples/python/text_skewness_correction.py

```diff
@@ -0,0 +1,53 @@
+'''
+Text skewness correction
+Usage:
+        python text_skewness_correction.py --image "Image path"
+'''
+
+import numpy as np
+import cv2 as cv
+
+
+def main():
+    import sys
+    import argparse
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-i", "--image", required=True, help="path to input image file")
+    args = vars(parser.parse_args())
+
+    # load the image from disk
+    image = cv.imread(cv.samples.findFile(args["image"]))
+    if image is None:
+        print("can't read image " + args["image"])
+        sys.exit(-1)
+    gray = cv.cvtColor(image, cv.COLOR_BGR2GRAY)
+
+    # threshold the image, setting all foreground pixels to
+    # 255 and all background pixels to 0
+    thresh = cv.threshold(gray, 0, 255, cv.THRESH_BINARY_INV | cv.THRESH_OTSU)[1]
+    coords = np.column_stack(np.where(thresh > 0))
+    angle = cv.minAreaRect(coords)[-1]
+
+    # the `cv.minAreaRect` function returns values in the
+    # range [-90, 0) if the angle is less than -45 we need to add 90 to it
+    if angle < -45:
+        angle = -(90 + angle)
+
+    # otherwise, just take the inverse of the angle to make
+    # it positive
+    else:
+        angle = -angle
```

Comment:
`angle` handling code is differ from C++ code.

Reply:
In python I have used  `coords = np.column_stack(np.where(thresh > 0))` where as in cpp I have used `findNonZero` due to which python code gives angle to be complimentary of the cpp code

Reply:
I believe it is better to use similar API calls (which produce similar results) to avoid Users confusion.
Please use `findNonZero()` in Python code too.